bundles/liberty/src/liberty/tools/ui/terminal/LocalDevModeLauncherDelegate.java

```diff
@@ -14,61 +13,33 @@
 public class LocalDevModeLauncherDelegate extends LocalLauncherDelegate {
 
     /**
-     * Id.
+     * LocalDevModeLauncherDelegate extension id.
      */
     public static final String id = "liberty.tools.ui.terminal.local.devmode.launcher.delegate";
```

Comment:
We can do a final passthrough at some point but we should qualify our stuff at least with prefix:  `io.openliberty.tools.*`